Python Tutorial Exercises(Week 5)/#7.py

```diff
@@ -0,0 +1,12 @@
+'''
+The total number of each car is obtained by adding the values of x, y, and z. (x[0]+y[0]+z[0]... and so on)
+Zip cars list and total_number list together
+print the final dictionary at the end
+'''
+cars = ["Hyundai Kona", "Kia Soul", "Mazda CX-30", "Volkswagen Taos", "BMW X1", "BMW X2"]
+old_prices = [22545, 20505, 23425, 24490, 36395, 37595]
+jan_increase = [540, 120, 600, 199, 1000, 3050]
+feb_increase = [153, 200, 123, 385, 450, 600]
+
+print({i : x for i, x in zip(cars,[x + y + z for x, y, z in zip(jan_increase, feb_increase, old_prices)])})
```

Comment:
You don't really need zip in this, you can just have
`print({cars[i]: old_prices[i]+jan_increase[i]+feb_increase[i] for i in range(len(cars))})`

Reply:
Now you can accomplish this without using a list comprehension first

Reply:
Wow... Didn't know you can use '+' like that. Good to know. Thanks for pointing that out.